pkg/project/resource_project_repo_test.go

```diff
@@ -114,3 +116,128 @@ func TestAccProjectRepo(t *testing.T) {
 		},
 	})
 }
+
+/*
+Test to assign large number of repositories to a project
+*/
+func TestAccAssignMultipleReposInProject(t *testing.T) {
+
+	const numRepos = 100
+	const repoNameInitial = "repo-"
+
+	name := "tftestprojects" + randSeq(10)
+	resourceName := "project." + name
+	projectKey := strings.ToLower(randSeq(6))
+
+	repoNamesStr := func(repoCount int) string {
+		var repoNames []string
+		for i := 0; i < repoCount; i++ {
+			repoNames = append(repoNames, repoNameInitial+strconv.Itoa(i))
+		}
+		j, err := json.Marshal(repoNames)
+		if err != nil {
+			return "[]"
+		} else {
+			return fmt.Sprint(string(j))
+		}
+	}
+
+	preCheck := func(t *testing.T, numRepo int) func() {
+		return func() {
+			testAccPreCheck(t)
+			for i := 0; i < numRepo; i++ {
+				createTestRepo(t, repoNameInitial+strconv.Itoa(i))
```

Comment:
This would be clearer using `fmt.Sprintf("%s%d")`

Reply:
improved